test/CMakeLists.txt

```diff
@@ -2,5 +2,4 @@
 add_executable(testregex testregex.cpp)
 add_test(testregexcorrect testregex)
 
-
 # correctness testing of checks
```

Comment:
Yes. Just say
`add_test(aliceo2namespace COMMAND check_clang_tidy.py options ${CMAKE_SOURCE_DIR}/test/aliceO2-namespace-naming.cpp)` or the like.